doc/bpdr/0003-set-criteria-for-static-site-framework.md

```diff
@@ -0,0 +1,59 @@
+# 3. Set Criteria for Recommended Static Site Generator
+
+## Status
+Proposed
+
+## Context
+TTS needs to build a lot of server-rendered websites, both for its own purposes and for other government agencies.
+
+Jekyll has been the popular go-to framework for this work, for a few reasons:
+- Many 18F employees were Rubyists
+- For a while, it had strong community and institutional support
+  - Github uses it as the engine for Github Pages
+- Federalist (now Cloud.gov Pages) supports it and USWDS built multiple themes for it, allowing teams to stand up a new government website very quickly
+- Markdown pages were relatively easier to edit by a wider variety of TTS employees than other frameworks
+- [TK - anything else?]
+
+In mid-2021 some core maintainers said Jekyll was [in hiatus](https://www.theregister.com/2021/09/14/future_of_jekyll_project_engine/) and recommended other frameworks to switch to. This hasn't been stated on the official website, and the [Jekyll Github repo](https://github.com/jekyll/jekyll) remains active, so Jekyll's future seems unclear.
+
+In addition, a number of TTS employees are unsatisfied with Jekyll because:
+- build times have become slower relative to other frameworks
+- [TK - what else?]
```

Comment:
- It’s difficult to support asset rendering (including building USWDS, a common component of our sites) in a way that supports far-futures cache headers. 
- Building locally in development is difficult to impossible without admin rights, hindering handoff to folks with less computer privileges.

Reply:
added!